packages/fxa-auth-server/tsconfig.json

```diff
@@ -20,7 +20,8 @@
     "noFallthroughCasesInSwitch": true,
 
     "types": ["node"],
-    "typeRoots": ["./types", "node_modules/@types"]
+    "typeRoots": ["./types", "node_modules/@types"],
+    "noImplicitAny": false
```

Comment:
😢

Reply:
Good point about this. I added it because my tests were failing when I tried importing a type def from `mozlog`, which I have custom typed in `types/mozlog/index.d.ts`. I can't find the definition. 

However when I switched away from `import { Logger } from 'mozlog'` to `const { Logger } = require('mozlog')`, this issue stopped.

Any idea why?

Reply:
Sorry, didn't notice this message until today.  Without looking into it, I don't know why that import wouldn't work.